models/contracts/contracts_project_name_mappings.sql

```diff
@@ -4,119 +4,27 @@
-    ,('kromatikafinance', 'Kromatika')
-    ,('kratosdao', 'Kratos Dao')
-    ,('curvefi', 'Curve')
-    ,('pika_perp', 'Pika Protocol')
-    ,('dhedge_v2', 'Dhedge')
-    ,('bitbtc', 'Bitbtc Protocol')
-    ,('teleportr', 'Teleportr/ Warp Speed')
-    ,('balancer_v2', 'Beethoven X')
-    ,('stargate', 'Stargate Finance')
-    ,('quixotic_v5', 'Quix')
-    ,('Unlock', 'Unlock Protocol')
-    ,('Xy Finance', 'XY Finance')
-    ,('Qidao', 'QiDao')
-    ,('Layerzero', 'Layer Zero')
-    ,('Xtoken', 'xToken')
-    ,('Instadapp', 'InstaDapp')
-    ,('Lifi', 'LiFi')
-    ,('Optimistic Explorer', 'Optimistic Explorer - Get Started NFT')
-    ,('ironbank', 'Iron Bank')
-    ,('iron_bank', 'Iron Bank')
-    ,('bluesweep', 'BlueSweep')
-    ,('hidden_hand', 'Hidden Hand')
-    ,('quixotic', 'Quix')
-    ,('project galaxy', 'Galxe')
-    ,('project_galaxy', 'Galxe')
-    ,('Masoud_ecc', 'ECC Domains')
-    ,('opx_finance', 'OPX Finance')
-    ,('pooltogether_v3', 'PoolTogether')
-    ,('beethovenx', 'Beethoven X')
-    ,('openxswap', 'OpenXSwap')
-    ,('eccdomains', 'ECC Domains')
-    ,('2pi_network','2Pi Network')
-    ,('twopi_network','2Pi Network')
-    ,('acryptos', 'AcryptoS')
-    ,('woofi', 'Woo Network')
-    ,('powerbomb_finance','Powerbomb Finance')
-    ,('powerbomb','Powerbomb Finance')
-    ,('lemma_finance','Lemma Finance')
-    ,('lemma','Lemma Finance')
-    ,('arrakis','Arrakis Finance')
-    ,('arrakis_finance','Arrakis Finance')
-    ,('collab_land_dao_pass', 'Collab.Land')
-    ,('Perpetualprotocol','Perpetual Protocol')
-    ,('perp_hottub','Perpetual Protocol')
-    ,('Collab Land', 'Collab.Land')
-    ,('collab_land', 'Collab.Land')
-    ,('Biconomy - Hyphen', 'Biconomy')
-    ,('angle', 'Angle Protocol')
-    ,('overnight', 'Overnight+')
-    ,('avt','AVT')
-    ,('Frax Finance', 'Frax')
-    ,('frax_finance', 'Frax')
-    ,('fraxfinance', 'Frax')
-    ,('DeFi Saver', 'DeFiSaver')
-    ,('defi_saver', 'DeFiSaver')
-    ,('Defisaver', 'DefiSaver')
-    ,('Decent', 'Decent.xyz')
-    ,('pika_tge', 'Pika Protocol')
-    ,('holograph_factory', 'Holograph')
-    ,('holograph_operator', 'Holograph')
-    ,('sound_xyz', 'Sound.xyz')
-    ,('sound xyz', 'Sound.xyz')
-    ,('splits','0xSplits')
-    ,('kyber', 'kyberswap')
-    ,('synthetix_v3', 'Synthetix')
-    ,('velodrome_v2', 'Velodrome')
-    ,('Maker','MakerDAO')
-    ,('Allo Protocol','Allo Protocol (Gitcoin)')
-    ,('pika_protocol_v4', 'Pika Protocol')
-    ,('summer_fi','Summer.fi')
-    ,('summerfi','Summer.fi')
-    ,('oasisapp','Summer.fi')
-    ,('niftykit_v2','NiftyKit')
-    ,('niftykit_v3','NiftyKit')
-    ,('cozy_v2_beta','Cozy Finance')
-    ,('cozy_v2_prod','Cozy Finance')
-    ,('highlight_xyz','Highlight.xyz')
-    ,('mint_fun','mint.fun')
-    ,('mintfun','mint.fun')
-    ,('allo_protocol', 'allo protocol (gitcoin)')
-    ,('union_protocol', 'Union Finance')
+{% set contracts_project_mapping_models = [
+  ref('contracts_optimism_project_name_mappings')
```

Comment:
This will be deprecated. Ideally all project names are mapped here so there's consistent naming across all chains.

Reply:
If needed, we can try to add logic for an 'exclude_chain' list. But the real ideal solve is for Dune decoding to enforce/override bad namespaces.

Reply:
>project names are mapped natively

What do you mean by this?

Reply:
sorry, I meant in the cross-chain version https://github.com/duneanalytics/spellbook/blob/main/models/_sector/contracts/contracts_project_name_mappings.sql